matminer/featurizers/structure.py

```diff
@@ -3612,3 +3613,189 @@ def citations(self):
     def implementors(self):
         return ["Alex Dunn", "Lauri Himanen"]
 
+    
+class GlobalInstabilityIndex(BaseFeaturizer):
+    '''
+    Will compute the global instability index of a structure.
+    
+    The default is to use IUCr 2016 bond valence parameters for computing 
+    bond valence sums. If the structure has disordered site occupancies 
+    or non-integer valences on sites, pymatgen's bond valence sum method 
+    can be used instead. 
+    
+    Note that pymatgen's bond valence sum method is prone to error unless 
+    the correct scale factor is supplied. A scale factor based on testing 
+    with perovskites is used here.
+    TODO: Use scipy to optimize scale factor for minimizing GII
+    
+    Based on the following publication:
+    
+    "Structural characterization of R2BaCuO5 (R = Y, Lu, Yb, Tm, Er, Ho, 
+        Dy, Gd, Eu and Sm) oxides by X-ray and neutron diffraction", 
+        A.Salinas-Sanchez, J.L.Garcia-Muñoz, J.Rodriguez-Carvajal,
+        R.Saez-Puche, and J.L.Martinez, Journal of Solid State Chemistry,
+        100, 201-211 (1992),
+        https://doi.org/10.1016/0022-4596(92)90094-C
+        
+    Args:
```

Comment:
Could you add a "Features:" block to the documentation that describes the feature being generated. See [PRDF](https://github.com/hackingmaterials/matminer/blob/4fc634830c032f12e477f4a9abe51a3411f4e99c/matminer/featurizers/structure.py#L281) as an example.